src/client/components/joinGame.jsx

```diff
@@ -4,13 +4,7 @@
 import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
 import {
-  Text,
-  Box,
-  Select,
-  FormControl,
-  Input,
-  FormLabel,
-  Button,
+  Text, Box, Select, FormControl, Input, FormLabel, Button,
```

Comment:
Is this the way linting should be? Because every time I run lint it tells me to format it the vertical way

Reply:
let's run a test this eve and then adjust if issues persist